### DISCUSSION TITLE
notifications

### DISCUSSION THREAD

**Discussion by aureliensas** [blakeblackshear/frigate#9843] [Ask A Question]
Hello,
I'm French, and I have a problem with Frigate notifications. I have 2 cameras on which ongoing events are available.
but I can only receive notifications from one camera. I don't get anything from the other
can you help me ?

https://codeshare.io/k0mwlO

(Frigate in Home Assistant VM)

**Comment by NickM-27** [comment]
what is your notification automatoin?

**Comment by aureliensas** [comment]
![IMG_7264](https://github.com/blakeblackshear/frigate/assets/83124554/dbb9c6e5-165e-405a-b794-c330e40f85a7)

I try all. It's ok for a camera "generic" but not work with Tenda
